packages/mdc-chips/_mixins.scss

```diff
@@ -429,6 +448,18 @@ $mdc-chip-ripple-target: ".mdc-chip__ripple";
     padding-left: $horizontal-padding-value;
     border-width: $width;
   }
+
+  $horizontal-margin: 0 - $horizontal-padding-value - $width;
+  $vertical-margin: 0 - $width;
+
+  #{$mdc-chip-ripple-target} {
+    @include mdc-feature-targets($feat-structure) {
+      top: $vertical-margin;
+      margin-right: $horizontal-margin;
```

Comment:
We do not need negative margin & transparent border if ripple is absolutely positioned inside chip.

We would need to explicitly set top & left position to `0` to start from top-left corner relative to its container.

Reply:
good point... you do need transparent borders, but do not need negative margins, my mistake.